repo.yml

```diff
@@ -0,0 +1 @@
+type: 'node'
```

Comment:
You no longer need this file and can remove it 🎉

Reply:
Thanks!